pkg/storage/tsdb/bucketcache/caching_bucket_test.go

```diff
@@ -196,7 +211,10 @@ func TestChunksCaching(t *testing.T) {
 					if i == 3 || i == 5 || i == 7 {
 						continue
 					}
-					cache.Delete(cachingKeyObjectSubrange(name, i*subrangeSize, (i+1)*subrangeSize))
+					err := cache.Delete(context.Background(), cachingKeyObjectSubrange(name, i*subrangeSize, (i+1)*subrangeSize))
```

Comment:
We use the `require` and `assert` packages elsewhere to do things like this.
```suggestion
					require.NoError(t, cache.Delete(context.Background(), cachingKeyObjectSubrange(name, i*subrangeSize, (i+1)*subrangeSize))
```

Reply:
Updated all. I am also not sure how this test become part of the PR. If I am not mistaken due to some lint error after I reabased.